scripts/run_gw.py

```diff
@@ -52,6 +52,14 @@ def log_to_pt(message, print_to_screen=False):
     superseded = False
     rc = None
 
+    # see https://bugzilla.mozilla.org/show_bug.cgi?id=1375514
+    # prevents g-w from never reaching termination condition when we do our supersede reruns and device issues occur
+    # https://github.com/taskcluster/generic-worker/blob/d3dda694d0031e8f1cd085f06c3b0f810321dac2/main.go#L485
+    gw_resolved_count_file = "tasks-resolved-count.txt"
```

Comment:
Do we know that the relative path will work? What is our working directory here?

Reply:
Peter made it seem like it would work in https://bugzilla.mozilla.org/show_bug.cgi?id=1375514.

It's what Wander shipped as a result of the discussion in that bug. https://hg.mozilla.org/build/puppet/rev/7cd4e03689ef.

Reply:
Great. I think that it is in the working directory of the generic-worker since that is how generic-worker writes it out. This seems ok to me, are you ready for a review?

Reply:
Yes please.